wger/exercises/templates/muscles/overview.html

```diff
@@ -9,26 +9,57 @@
 
 {% block header %}
 <script src="{% static 'js/exercises.js' %}"></script>
+<script src="{% static 'js/svg.js' %}"></script>
 
 <script>
-$(document).ready(function() {
-    //Prefetch the background images
-    {% for muscle in muscle_list|first %}
-        wgerPrefetchImages(['/static/images/muscles/main/muscle-' + {{muscle.id}} + '.svg']);
+    $(document).ready(function () {
+        //Prefetch the background images
+        {% for muscle in muscle_list|first %}
+        wgerPrefetchImages(['/static/images/muscles/main/muscle-' + {{ muscle.id }} + '.svg']);
     {% endfor %}
     wgerPrefetchImages(['/static/images/muscles/muscular_system_back.svg']);
-
+    $('#muscle-systems').append(front_muscles);
+    $('#muscle-systems').append(back_muscles);
 
     // Hightlight the first element
     wgerHighlightMuscle($('.muscle').first());
 
     // Highlight on hover
-    $('.muscle').hover(function(e) {
+    $('.muscle').hover(function (e) {
         e.preventDefault();
         wgerHighlightMuscle(this);
+        var is_front = $(this).data('isFront');
```

Comment:
You could consider using ES6 specification instead of ES5 in the variable declaration. Consider going through this:
 https://www.tutorialspoint.com/es6/es6_variables.htm